grammars/puppet.cson

```diff
@@ -146,7 +146,7 @@
     'include': '#variable'
   }
   {
-    'begin': '(?i)\\b(import|include|contain)\\b\\s*'
+    'begin': '(?!.*=>)\\b(import|include|contain|require)\\s+'
```

Comment:
I would prefer having it at the end of the regex as that would more clearly imply that => is not allowed *after* the keywords.